ApiGateway/src/main/java/app/controllers/StockPriceRecordController.java

```diff
@@ -0,0 +1,22 @@
+package app.controllers;
+
+import app.services.StockPriceRecordService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin(maxAge = 3600)
+@RestController
+@RequestMapping("/api")
+public class StockPriceRecordController {
+
+    @Autowired
+    private StockPriceRecordService stockPriceRecordService;
+
+    @GetMapping("/stocks/refresh")
+    public void refresh() {
+        stockPriceRecordService.refresh();

```

Comment:
The reason I'm adding this is because on the day of the presentation, if we choose to deploy and use the program in prod, we will have to add a way to delete all the Stock prices. Otherwise, there may be too much load on the server as other people are storing the prices in mysql too.